src/app/modules/group/components/user-list/user-list.component.ts

```diff
@@ -8,43 +10,48 @@ import { GetGroupMembersService, Member } from '../../http-services/get-group-me
   templateUrl: './user-list.component.html',
   styleUrls: [ './user-list.component.scss' ]
 })
-export class UserListComponent implements OnChanges, OnDestroy {
+export class UserListComponent implements OnInit {
 
   @Input() group? : Group;
   state: 'loading' | 'error' | 'empty' | 'ready' = 'loading';
+  currentSort: string[] = [];
 
   members: Member[] = [];
 
-  private subscription?: Subscription;
+  private dataFetching = new BehaviorSubject<'Reload'|'Sort'>('Reload');
 
   constructor(private getGroupMembersService: GetGroupMembersService) { }
 
-  ngOnChanges(_changes: SimpleChanges): void {
-    this.reloadData();
+  ngOnInit(): void {
+    this.dataFetching.pipe(
+      switchMap(mode => {
+        if (this.group) {
+          if (mode === 'Reload')
+            this.state = 'loading';
+
```

Comment:
"Loading" while it is currently fetching the loading would make sense as well, but not via hiding the whole table. I see that the primeng table has a `loading` property that could be used for that.

Reply:
fixed